docs/pages/what-is-fable-server.md

```diff
@@ -0,0 +1,9 @@
+# Fable Server
+When you build your Fable app or run the app on developement mode, a Fable server will run in the background. This server will work and orchestrate the compilation process with webpack. 
+
+At first you have the `fable-loader`. A loader in webpack tells webpack how to read and `require` certain types of files. In the case of `fable-loader`, it tells webpack how to deal with files of type `*.fsproj`, `*.fs` or `*.fsx`. Namely by sending these files off to the Fable server to compile them. Because the Fable server only compiles F# to Babel, a `babel-loader` will take care of the output of Fable to generate the eventual javascript output that runs on your browser or on Node.
```

Comment:
Fable doesn't call babel-loader but babel-core. Maybe we could rewrite the last sentence as:

"Because Fable only generates an AST (Abstract Syntax Tree), Babel will take care of the output to generate the eventual javascript code that runs on your browser or on Node."

Also adding a note at the end:

> The Fable daemon can communicate with other tools besides Webpack. There's also a Rollup plugin (link: https://www.npmjs.com/package/rollup-plugin-fable) available if you prefer to bundle your JS code with Rollup instead of Webpack.